src/metabase/api/dataset.clj

```diff
@@ -52,8 +52,7 @@
    & {:keys [context export-format was-pivot]
       :or   {context       :ad-hoc
              export-format :api}}]
-  (span/with-span!
-    {:name "run-query-async"}
+  (span/with-span! {:name "run-query-async"}
```

Comment:
I cannot figure out how to make Emacs indent this like `:defn` in `.dir-locals.el`, as something named `with-` it wants to indent it as `1`. So I moved the map up onto the first line so it stopped trying to indent this all crazy.